distributions/aws/test/canary/scripts/run_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# All the inputs to this script as environment variables 
+# REPO_PATH : Local root path of the kubeflow-manifest github repo 
+# CLUSTER_NAME : Name of the EKS cluster
+# CLUSTER_REGION : Region of the EKS cluster
+
+
+# Script configuration
+set -euo pipefail
+
+function onError {
+  echo "Run test FAILED. Exiting."
+}
+trap onError ERR
+
+export CANARY_TEST_DIR=${REPO_PATH}/distributions/aws/test/canary
+export E2E_TEST_DIR=${REPO_PATH}/distributions/aws/test/e2e
+
+# Connect to eks cluster 
+aws eks update-kubeconfig --name $CLUSTER_NAME --region $CLUSTER_REGION
+
+# Modify metadeta file
+cd $CANARY_TEST_DIR
+sed -i 's/$CLUSTER_NAME/'"$CLUSTER_NAME"'/g' metadata-canary
+sed -i 's/$CLUSTER_REGION/'"$CLUSTER_REGION"'/g' metadata-canary
+
+mkdir -p $E2E_TEST_DIR/.metadata/
+cp metadata-canary $E2E_TEST_DIR/.metadata/
```

Comment:
Can we not copy the `metadata-canary` file over first and then edit so we don't edit the source ?

Reply:
@akartsky instead of maintaining a separate file and using sed command for each variable, what do you think about using `printf` command directly to write the contents to the file

```
printf '{
"region": '$CLUSTER_REGION'
}' > .metadata/canary-meta.json
```

Reply:
@mbaijal 
It's happening inside the docker image so doesn't matter if we edit first or if we copy over and then edit 

@surajkota 
By having a dummy metadata file it's easier to see the skeleton of the file and even reuse it for local testing for example

Reply:
approving since this can be changed later if there are too many vars